apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -75,7 +76,9 @@ function VerticalTimeline({ status }: VerticalTimelineProps) {
 				/>
 				Sign waiver
 			</li>
-		) : status === PortalStatus.waived || status === PortalStatus.confirmed ? (
+		) : status === PortalStatus.waived ||
+		  status === PortalStatus.confirmed ||
+		  status === PortalStatus.attending ? (
```

Comment:
Rather than fighting the linters, could we extract some of these conditions to named boolean values, e.g. `attendanceConfirmed` and `signedWaiver`